app/src/main/java/com/endava/myendava/listeners/OnProfileEditListener.java

```diff
@@ -0,0 +1,6 @@
+package com.endava.myendava.listeners;
+
+public interface OnProfileEditListener {
```

Comment:
usually in interfaces we use the past tense ... OnProfileEdit**ed** and onEditClick**ed**, since the action is already performed. using the present suggest that the action is happening now or it will be happening shortly